remix/app/routes/_app._index.tsx

```diff
@@ -1,17 +1,32 @@
-import { User } from '@prisma/client';
-import { useOutletContext } from '@remix-run/react';
+import { useLoaderData } from '@remix-run/react';
 
 import AppLink from '../components/AppLink';
+import NewsletterRepository from '../repositories/newsletter.server';
+
+export async function loader() {
+  const newsletters = await NewsletterRepository.findMany({});
```

Comment:
The newsletters should belong to the current user 🙈 So the where condition must have a user ID.

Reply:
I thought that all newsletters can view by anyone 😅.

Added here: [6c87ca5](https://github.com/nimblehq/js-framework-benchmark/pull/122/commits/6c87ca51eb41620e3ead08669d89eb10f54cb622)